resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,1981 @@
+        '406':
+          description: 'Not Acceptable'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/errorResponse'
+                - example:
+                    code: ACCEPT_HEADER_INVALID
+              examples:
+                badRequest:
+                  description: The accept header is missing or invalid.
+                  value:
+                    code: ACCEPT_HEADER_INVALID
+      deprecated: false
+      security:
+        - userRestricted:
+            - common-transit-convention-traders
+
+  /customs/transits/movements/arrivals/{arrivalId}/messages:
+    post:
+      tags:
+      - common-transit-convention-traders
+      summary: Send a message related to an arrival
+      description: >
+        Send a message to the Customs Office of Destination about an arrival.
+        
+        For example, if you are a Trader at Destination, you will send an ‘Unloading Remarks’ E_ULD_REM (IE044) message to the Customs Office of Destination indicating that unloading of the goods has been completed.
+        
+        A successful response will contain a URI and an arrival ID for the message, which you can use to retrieve the message status.
+
+        **Note:** If an Economic Operators Registration and Identification (EORI) number has not been assigned to you, an HTTP 403 Forbidden error is returned.
+      operationId: SendAMessageRelatedToAnArrival
+      parameters:
+        - $ref: '#/components/parameters/contentTypeHeader'
+        - in: path
+          name: arrivalId
+          description: The arrival ID specifying the arrival to return.
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '1'
+      responses:
+        '202':
+          description: 'Accepted'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/CustomsTransitsMovementsArrivalsMessagesResponse'
+                - description: JSON payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/arrivals/1/messages/2
+                      arrival:
+                        href: /customs/transits/movements/arrivals/1
+                    arrivalId: '1'
+                    messageId: '2'
+                    messageType: IE044
+                    body: <CC044A>...</CC044A>
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/arrivals/1/messages/2
+                  arrival:
+                    href: /customs/transits/movements/arrivals/1
+                arrivalId: '1'
+                messageId: '2'
+                messageType: IE044
+                body: <CC044A>...</CC044A>

```

Comment:
Yet again, Redocly linter flagged absence of a `4XX` response for this operation.